pkg/operator/ceph/cluster/osd/volumes.go

```diff
@@ -86,7 +87,26 @@ func getDeviceMapperVolume() (v1.Volume, v1.VolumeMount) {
 	return volume, volumeMounts
 }
 
-func getPVCOSDVolumes(osdProps *osdProperties) []v1.Volume {
+func getDataBridgeVolumeSource(claimName, configDir, namespace string, inProvisioning bool) v1.VolumeSource {
+	var source v1.VolumeSource
+	if inProvisioning {
+		source.EmptyDir = &v1.EmptyDirVolumeSource{
```

Comment:
If I understand correctly, the osd prepare volumes will remain unchanged (the memory volumeSource is just refactored here), and the osd daemon volume will now have a hostpath.

Reply:
That's right.